changelogs/fragments/81450-list-filters.yml

```
@@ -0,0 +1,4 @@
+bugfixes:
+  - "ansible-console - fix filtering by collection names when a collection search path was set (https://github.com/ansible/ansible/issues/81450)."
+minor_changes:
+  - "ansible-doc - allow to filter listing of collections and metadata dump by more than one collection (https://github.com/ansible/ansible/issues/81450)."
```

`https://github.com/ansible/ansible/issues/81450` redirects to `https://github.com/ansible/ansible/pull/81450` but better to use the right URL in the first place

```suggestion
  - "ansible-console - fix filtering by collection names when a collection search path was set (https://github.com/ansible/ansible/pull/81450)."
minor_changes:
  - "ansible-doc - allow to filter listing of collections and metadata dump by more than one collection (https://github.com/ansible/ansible/pull/81450)."
```